scripts/bamboo-release.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Expects env var SITE to be one of zl, ces, etc.
+
+# Intended to be run from Bamboo.
+# - Takes built pih-spa zip and moves it into /var/www/html for distribution
+# - Adds a VERSION file
+# - Updates the 'latest' package
+#
+# Releases have names like `pih-spa-SITE-XX-YY.zip` where
+# - SITE is the site name
+# - XX is the spa-frontend commit number in master
+# - YY is the config repo commit number in master
```

Comment:
I was wondering what you meant by "commit number" because I didn't think we'd want to use a commit hash for this because we'd need something sequential for ease of use... but then I see that you use "git rev-list --count" below which appears to give the count of commits to date which we use as a "commit number". I don't have experience with "rev-list" but if you think that's stable enough to use, let's go with that (for example, what if a build is manually retriggered with no change in the commit history)... you should have access to the bamboo build number if that's works better (I believe that is what we use for the debian packages)... though I making it independent of bamboo isn't the worst thing in the world...

Reply:
If the job reruns and there are no new commits in either repo, the package will be overwritten. Can you think of a situation where that could be a problem?

Reply:
No, just double-checking.